lfisuite.py

```diff
@@ -5,42 +5,45 @@
 # Tutorial Video: https://www.youtube.com/watch?v=6sY1Skx8MBc
 # Github Repository: https://github.com/D35m0nd142/LFISuite
 
-#!/usr/bin/python
+# !/usr/bin/python
```

Comment:
```suggestion
#!/usr/bin/env python3
```
standard way to define a shebang.